### PR TITLE
Improve tests, requirements

### PR DESCRIPTION
**Pull request by khvn26**
- Removed `tests/__init__.py` to improve discoverability
 - Removed `tests/test_benchmark.py`: benchmarks are not unit tests, undesired dependency on `statsd` library

### CODE REVIEW

**Comment by vitawasalreadytaken** [comment]
Hi, thanks for the PR! I am happy to remove `tests/__init__.py` but we definitely want to keep the benchmark since the whole point of this library is to be (verifiably) faster than `statsd`. The dependency on `statsd` only exists in `dev-requirements.txt` so it does not affect users of this library.

**Comment by khvn26** [comment]
Hello. Thanks for your reply. 

To ensure service health, our CI builds run tests for some of our dependencies along with our own. I don't think that it's sensible to install all of the dev dependencies, along with a client library we're not using, and run benchmarks on every build.

I'd consider one of the following:
 - skip benchmark tests by default (or if `statsd` is not installed)
 - factor test and benchmark dependecies in two separate files (or import `statsd` only if `--skip-benchmarks` flag is not provided to `pytest`)

Let us choose an approach and I'll update my PR accordingly.

**Comment by vitawasalreadytaken** [comment]
I like the idea of skipping benchmarks if `statsd` is not importable. pytest has a built-in function for that: https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency

Thanks!

**Comment by khvn26** [comment]
Any updates?

File: requirements.txt
Comment (vitawasalreadytaken) [inline requirements.txt]: I have had a lot of painful experiences with open-ended dependencies like these. Sooner or later a version will come out that's not backwards compatible and new builds suddenly start failing.

I am up for updating `cython` to the newest current version, but pinning it with `==`. Since the major version is still `0` it is possible that future minor versions will break compatibility according to semver rules.

`ql-cystatsd` is our fork of https://github.com/scivey/cystatsd that will hopefully cease to exist once some PRs are merged upstream. So I can't make any guarantees about its future versions either.

**Review by vitawasalreadytaken — CHANGES_REQUESTED** [review]
Hey, sorry for the delay. I am happy to merge if we keep the dependencies safely pinned :)

File: requirements.txt
Comment (khvn26) [reply requirements.txt]: I have pinned the Cython version.

VCS dependencies can't be pinned, unless a specific version branch was checked out. Either do that or, if you don't plan to introduce further changes, I suggest leaving the `ql-cystatsd` dependency as it is.

**Review by vitawasalreadytaken — APPROVED** [review]

**Comment by vitawasalreadytaken** [comment]
Thanks again, and sorry this took so long.

**Pull request merged**
